wagtail/embeds/migrations/0009_convert_null_to_empty_string.py

```diff
@@ -0,0 +1,18 @@
+from django.db import migrations
+
+
+def migrate_forwards(apps, schema_editor):
+    Embed = apps.get_model("wagtailembeds.Embed")
+
+    Embed.objects.filter(thumbnail_url__isnull=True).all().update(thumbnail_url='')
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtailembeds", "0008_allow_long_urls"),
+    ]
+
+    operations = [
+        migrations.RunPython(migrate_forwards, migrations.RunPython.noop),
+    ]
```

Comment:
Hi @cnk. Thank you very much for this, but I think this migration might be redundant. I could have probably done a better job of outlining that we would only need this if adding `null=True` back to the field... The AlterField operation in the previous migration should have already taken care of converting NULL values to blank strings.